templates/openio.pp.j2

```diff
@@ -1,4 +1,7 @@
-class {'openiosds':}
+class {'openiosds':
+uid     => '{{ openio_user_openio_uid }}',
```

Comment:
These should be declared in role defaults as follows:
```
openio_user_openio_uid: 120
openio_user_openio_gid: 220
```

Reply:
il est deja en 120/220

Reply:
Ils sont déjà à 120/220 dans le Puppet, il faut ajouter les variables par défaut dans le `defaults/main.yml` pour ne pas avoir à les définir obligatoirement dans le playbook du côté Ansible.